xbmc/utils/SeekHandler.cpp

```diff
@@ -182,6 +182,33 @@ void CSeekHandler::Seek(bool forward, float amount, float duration /* = 0 */, bo
   m_timer.StartZero();
 }
 
+void CSeekHandler::SeekSeconds(int seconds)
+{
+  if (seconds == 0)
+    return;
+
+  m_requireSeek = true;
+  g_infoManager.SetSeeking(true);
+  m_seekStep = 0;
+  m_analogSeek = false;
+  m_seekDelay = 0;
+
+  if (g_infoManager.GetTotalPlayTime())
```

Comment:
is this meant to check that we have a non-zero playtime or that we're actually playing something? either make explicit what you mean or use an appropriate method.